CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2019-01-18
+### Unreleased
+- Retry calls to Stellar horizon API regardless of the type of error code it receives.
+- Add `StellarSpectrum::Log` logger class.
```

Comment:
Since this is the changelog, it'll be read by other developers when upgrading. I think it should be more about the new behaviour than the classes, when possible

```suggestion
- Log, if logger is set, when exceptions occur and payment is retried
```

Reply:
Got it!